src/lang/Messages.properties

```diff
@@ -2379,6 +2379,12 @@ view.options.label.fontName					= Font Name:
 view.options.label.fontSize					= Font Size:
 view.options.label.largeRequestSize			= Large request view min size:
 view.options.label.largeResponseSize		= Large response view min size:
+
+view.options.label.lookAndFeel              = Look and Feel: 
```

Comment:
I know the properties files aren't consistent, but with newer content we are trying to stick with full lower case on the key names.